components/application-mgt/org.wso2.carbon.identity.application.mgt/src/test/resources/dbscripts/identity.sql

```diff
@@ -0,0 +1,1191 @@
+
+        PRIMARY KEY (ID));
+
+ALTER TABLE SP_APP ADD CONSTRAINT APPLICATION_NAME_CONSTRAINT UNIQUE(APP_NAME, TENANT_ID);
+ALTER TABLE SP_APP ADD CONSTRAINT APPLICATION_UUID_CONSTRAINT UNIQUE(UUID);
+
+CREATE TABLE IF NOT EXISTS SP_METADATA (
+            ID INTEGER AUTO_INCREMENT,
+            SP_ID INTEGER,
+            NAME VARCHAR(255) NOT NULL,
+            VALUE VARCHAR(255) NOT NULL,
+            DISPLAY_NAME VARCHAR(255),
+            TENANT_ID INTEGER DEFAULT -1,
+            PRIMARY KEY (ID),
+            CONSTRAINT SP_METADATA_CONSTRAINT UNIQUE (SP_ID, NAME),
+            FOREIGN KEY (SP_ID) REFERENCES SP_APP(ID) ON DELETE CASCADE);
+
+CREATE TABLE IF NOT EXISTS SP_INBOUND_AUTH (
+            ID INTEGER NOT NULL AUTO_INCREMENT,
+            TENANT_ID INTEGER NOT NULL,
+            INBOUND_AUTH_KEY VARCHAR (255),
+            INBOUND_AUTH_TYPE VARCHAR (255) NOT NULL,
+            INBOUND_CONFIG_TYPE VARCHAR (255) NOT NULL,
+            PROP_NAME VARCHAR (255),
+            PROP_VALUE VARCHAR (1024) ,
+            APP_ID INTEGER NOT NULL,
+            PRIMARY KEY (ID));
+
+ALTER TABLE SP_INBOUND_AUTH ADD CONSTRAINT APPLICATION_ID_CONSTRAINT FOREIGN KEY (APP_ID) REFERENCES SP_APP (ID) ON DELETE CASCADE;
+
+CREATE TABLE IF NOT EXISTS SP_AUTH_STEP (
+            ID INTEGER NOT NULL AUTO_INCREMENT,
+            TENANT_ID INTEGER NOT NULL,
+            STEP_ORDER INTEGER DEFAULT 1,
+            APP_ID INTEGER NOT NULL ,
+            IS_SUBJECT_STEP CHAR(1) DEFAULT '0',
+            IS_ATTRIBUTE_STEP CHAR(1) DEFAULT '0',
+            PRIMARY KEY (ID));
+
+ALTER TABLE SP_AUTH_STEP ADD CONSTRAINT APPLICATION_ID_CONSTRAINT_STEP FOREIGN KEY (APP_ID) REFERENCES SP_APP (ID) ON DELETE CASCADE;
+
+CREATE TABLE IF NOT EXISTS SP_FEDERATED_IDP (
+            ID INTEGER NOT NULL,
+            TENANT_ID INTEGER NOT NULL,
+            AUTHENTICATOR_ID INTEGER NOT NULL,
+            PRIMARY KEY (ID, AUTHENTICATOR_ID));
+
+ALTER TABLE SP_FEDERATED_IDP ADD CONSTRAINT STEP_ID_CONSTRAINT FOREIGN KEY (ID) REFERENCES SP_AUTH_STEP (ID) ON DELETE CASCADE;
+
+CREATE TABLE IF NOT EXISTS SP_CLAIM_DIALECT (
+	   	ID INTEGER NOT NULL AUTO_INCREMENT,
+	   	TENANT_ID INTEGER NOT NULL,
+	   	SP_DIALECT VARCHAR (512) NOT NULL,
+	   	APP_ID INTEGER NOT NULL,
+	   	PRIMARY KEY (ID));
+
+ALTER TABLE SP_CLAIM_DIALECT ADD CONSTRAINT DIALECTID_APPID_CONSTRAINT FOREIGN KEY (APP_ID) REFERENCES SP_APP (ID) ON DELETE CASCADE;
+
+CREATE TABLE IF NOT EXISTS SP_CLAIM_MAPPING ( 
+	   	ID INTEGER NOT NULL AUTO_INCREMENT, 
+	   	TENANT_ID INTEGER NOT NULL, 
+	   	IDP_CLAIM VARCHAR (512) NOT NULL , 
+	   	SP_CLAIM VARCHAR (512) NOT NULL , 
+	   	APP_ID INTEGER NOT NULL, 
+	   	IS_REQUESTED VARCHAR(128) DEFAULT '0', 
+		IS_MANDATORY VARCHAR(128) DEFAULT '0', 
+	   	DEFAULT_VALUE VARCHAR(255), 
+	   	PRIMARY KEY (ID));
+
+ALTER TABLE SP_CLAIM_MAPPING ADD CONSTRAINT CLAIMID_APPID_CONSTRAINT FOREIGN KEY (APP_ID) REFERENCES SP_APP (ID) ON DELETE CASCADE;
```

Comment:
Line exceeds the 120 character limit

Reply:
Fixed with [48da372](https://github.com/wso2/carbon-identity-framework/pull/3494/commits/48da3722b0215cc4eb8f430b71d21a89731805ba)